src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -5265,6 +5265,7 @@ namespace InternalHeatGains {
             e.HWPower = 0.0;
             e.SteamPower = 0.0;
             e.BaseHeatPower = 0.0;
+            e.CO2Rate = 0.0;
```

Comment:
This was forgotten here. Note that not all variables in `ZnRpt` are set to zero here. Just the ones that are accumulated with `+=` in this function.